app/apps/camunda/services.py

```diff
@@ -31,25 +31,28 @@ def _process_request(self, request_path, request_body=None, post=False):
                 )
             return response
         except requests.exceptions.Timeout:
-            return Response(
+            response = Response(
                 "Camunda service is offline",
                 status=status.HTTP_503_SERVICE_UNAVAILABLE,
             )
+            response.ok = False
+            return response
         except requests.exceptions.RequestException:
-            return Response(status=status.HTTP_400_BAD_REQUEST)
+            response = Response(status=status.HTTP_400_BAD_REQUEST)
+            response.ok = False
```

Comment:
@xavierbloemen heel hacky, maar ik heb het maar even zo opgelost.

Reply:
Loop er vandaag ook tegen aan. Voor nu okay-ish en we verzinnen er wel wat beters op